crates/fiber/src/lib.rs

```diff
@@ -96,7 +118,7 @@ pub unsafe trait RuntimeFiberStackCreator: Send + Sync {
     ///
     /// This is useful to plugin previously allocated memory instead of mmap'ing a new stack for
     /// every instance.
-    fn new_stack(&self, size: usize) -> Result<Box<dyn RuntimeFiberStack>, Error>;
+    fn new_stack(&self, size: usize) -> core::result::Result<Box<dyn RuntimeFiberStack>, Error>;
```

Comment:
A nice trick to avoid `core::result::Result` here is to, in each module, change:

```rust
type Result<T> = some::Result<T>;
```

to 

```rust
type Result<T, E = some::Error> = core::result::Result<T, E>;
```

That way you can use any `Result` type-alias as the actual real `Result` without worry

Reply:
Done!